examples/cargo-make/scripts/web-report.sh

```diff
@@ -117,13 +131,14 @@ function print_footer {
     c="${BOLD}${YELLOW}C${RESET} = Cucumber Test Runner"
     d="${BOLD}${YELLOW}F${RESET} = Fantoccini WebDriver"
     l="${BOLD}${YELLOW}L${RESET} = Cargo Leptos"
+    m="${BOLD}${RED}M${RESET} = Cargo Leptos Medata Only (${ITALIC}ci is not configured to build with cargo-leptos${RESET})"
```

Comment:
Small thing but I think `Medata` is probably a typo from `Metadata` here?

Reply:
Could you please commit these changes with the typo?

I will push another PR with some doc improvements.